_team/will-flowers.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Will Flowers
+layout: team
+job_title: Web Developer Intern
+job_description: Will is a novice web developer at Savas, learning from a growing team and contributing to a variety of clients.
+photo: "assets/img/team/cards/will-card.jpg"
+card: "assets/img/team/will-flowers.jpg"
```

Comment:
@flowersw It looks like these are reversed. The card image should be the wide hobby image. So, you'll need to switch the filenames of the images, then switch the photo and card image urls in lines 6 and 7 of will-flowers.md. I'm not sure if i explained that well, so let me know if it doesn't make sense!